examples/js/scatterplot.js

```diff
@@ -754,4 +754,38 @@ function Scatterplot(el, component, svgClass) {
     return str.replace(' ', '').toLowerCase()
   }
 }
+
+  var scatterplot1Tab1 = document.querySelector('.scatterplot-1-tab-1');
```

Comment:
Sorry for sending you a bad example. However, now you know how to do the hover transition with only css. So try keep using css for those transitions, but just trigger the transitions from the tablist classname during tab onClicks.

Also, Josh is right about those labels.